gst/nnstreamer/tensor_aggregator/tensor_aggregator.c

```diff
@@ -427,7 +427,9 @@ gst_tensor_aggregator_sink_event (GstPad * pad, GstObject * parent,
       break;
     }
     case GST_EVENT_FLUSH_STOP:
-      gst_tensor_aggregator_reset (self);
+      if (self->adapter) {
```

Comment:
As we discussed, let's consider to init/free the config in _init()/_finalize() function.

Reply:
Updated, Plz check.